### PR TITLE
Serve images in Modern Formats

### PR DESCRIPTION
**Pull request by markus-ntt-seidl**
I converted the #67 to this. By doing this I tried to stay away a bit from mentioning formats as "WebP" and/or "AVIF" being the final/goto format. 
There seem to be a lot of new formats besides WebP/AVIF and it's not clear to me which one is the "winner". Also some of them provide additional features. 

I think this pattern should only be a hint that image formats should be researched by the dev team and not "just use jpeg".

### CODE REVIEW

File: docs/catalog/web/serve-images-in-modern-formats.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+---
+version: 1.0
+submitted_by: dubrie
+published_date: TBD
+category: web
+tags: 
+ - image
+ - size:small
```
Comment (dubrie) [inline docs/catalog/web/serve-images-in-modern-formats.md:8]: please add `role:web-developer` has a tag here

File: docs/catalog/web/serve-images-in-modern-formats.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+## Description
+
+Images are one of the main parts of modern Websites and used almost on every website. Modern Image formats can help to reduce bandwith, storage and computing requirements on the displaying device.
+
+There are serveral new image formats to choose from, for example WebP and AVIF. All Formats provide better compression ratio, increased quality and more features as traditional formats.
+
+
+## Solution
+
+Validate which image format is supports the use case of the web application the best. Convert images to modern image formats and deliver them to the displaying devices.
```
Comment (dubrie) [inline docs/catalog/web/serve-images-in-modern-formats.md:22]: remove extra `is` in this sentence.

Comment (dubrie) [inline docs/catalog/web/serve-images-in-modern-formats.md:22]: Is there any guidance on how to validate image formats?

File: docs/catalog/web/serve-images-in-modern-formats.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+`SCI = (E * I) + M per R`  
+[Sofware Carbon Intensity Spec](grnsft.org/sci)
+
+Concerning the SCI equation, optimizing peak CPU utilization will impact two parts:
+
+- `E`: Reducing the amount of emissions by reducing time spent on transfering and processing images
+
+## Assumptions
+- The ability to use modern webbrowsers with supportes modern image formats
```
Comment (dubrie) [inline docs/catalog/web/serve-images-in-modern-formats.md:35]: replace `with` with `which`

**Review by dubrie — CHANGES_REQUESTED** [review]
Some minor format and typo changes requested for initial review

File: docs/catalog/web/serve-images-in-modern-formats.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
+---
+version: 1.0
+submitted_by: dubrie
```
Comment (dubrie) [inline docs/catalog/web/serve-images-in-modern-formats.md:3]: Happy to take credit, but this should be your own github alias :-)

Comment (markus-ntt-seidl) [reply docs/catalog/web/serve-images-in-modern-formats.md:3]: To be honest I was unsure if this is the issuer (as I'm "only" converting it) or the convertor of the issue :)

File: docs/catalog/web/serve-images-in-modern-formats.md (hunk 2 above)
Comment (markus-ntt-seidl) [reply docs/catalog/web/serve-images-in-modern-formats.md:22]: Yes. At least: validate if image size actually decreases (one could have the worst case of images in the project), if the target browser supports the new format and if the image quality is as expected.
On the other side we are not targeting specific applications. So I thought that a project which, for example, implements the next google earth would have many more requirements (which I can not think of). 

Do you think we should state a few validation points as guideline? Most of the patterns I saw where really short, so I converted them to short (1, 2 or 3 sentences) patterns as well.

**Review by dubrie — APPROVED** [review]
This looks great, thanks for the submission!

**Pull request merged**